cli/module_generate/scripts/generate_stubs.py

```diff
@@ -19,13 +26,19 @@ def main(
 
     module_name = f"viam.{resource_type}s.{resource_subtype}.{resource_subtype}"
     module = import_module(module_name)
-    resource_name = "".join(word.capitalize() for word in resource_subtype.split("_"))
+    if resource_subtype == "input":
+        resource_name = "Controller"
+    elif resource_subtype == "slam":
+        resource_name = "SLAM"
+    elif resource_subtype == "mlmodel":
+        resource_name = "MLModel"
+    else:
+        resource_name = "".join(word.capitalize() for word in resource_subtype.split("_"))
     resource = getattr(module, resource_name)
     methods = inspect.getmembers(resource, predicate=inspect.isfunction)
 
     imports = []
-    abstract_methods = []
-    for name, method in methods:
+    for _, method in methods:
```

Comment:
Do we still need this? Can we get all the imports we need from the AST?

Reply:
Yeah, this is something I was wondering and wanted to take out.
AST doesn't seem to show where the types are from. So for example, `Vector3` and `GeoPoint` don't tell us if it's from `_pb2` module or not. But I'm not sure how useful this is in the first place because from the limited testing I've done, I saw that these types are usually exported in the `__init__.py` file so it would be included when we `from <component> import *`.

Reply:
For now I'll take it out and see what happens